### ISSUE TITLE
Dependency on core-js@3 breaks Nuxt build

### ISSUE THREAD

**Issue by Zsavajji** [sampotts/plyr#1414]
Hi, as title says, i cannot upgrade plyr to 3.5.3 in my nuxt project because of the updated core-js dependency. Nuxt decided to stick to core-js@2 (https://github.com/nuxt/nuxt.js/issues/5313) and updating plyr breaks the build.
Are there any real advantages in moving to core-js@3?

**Comment by sampotts** [comment]
Yep, there from what I can see... https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md

Can't you fix the version in your package.json in `resolutions` ?

**Comment by clarkdo** [comment]
@Zsavajji Thanks for using Nuxt.js.

Nuxt.js has provided a way to use `core-js@3` since 2.6.0 as [release notes](https://nuxtjs.org/guide/release-notes#core-js) described. 

But one thing needs to be notified is that you need to make sure that all of your dependencies support `core-js@3` like `plyr`, because if `core-js@2` and `core-js@3` are both used, it may lead to unexpected error due to uncertain `core-js` version in your node_modules which are hoisted by npm/yarn.

**Comment by mikemellor11** [comment]
Just out of interest, why is plyr shipped with core-js as a dependancy? If all the js files are bundled before the package is published whats the need for the polyfills and core in the deps and not devDeps?
